### ISSUE TITLE
Session median not accurate due to DNFs

### ISSUE THREAD

**Issue by GregoryCannon** [CubeStuffs/CubeTime#164]
The session mean and median are not accurate because they exclude DNFs. The mean makes sense because it's not clear what time to use for the DNFed solve, but for median it is very possible to include them in the calculation.

**Comment by pdtxie** [comment]
Yeah I was thinking of adding a toggle to switch between including and excluding DNFs in mean calculation. As for the median, definitely an oversight on my side. Will include DNFs in next update.
